eberron.md

```diff
@@ -213,7 +222,7 @@ Add another steed summoned for every level cast above 1st.
 - **Duration:** instant
 - **Damage Type:** Piercing
 
-Magically shape a small piece of wood into a javelin. The javelin will then be thrown with wind magic at the chosen target. Make a spellcasting roll against the target's armor. If the attack succeeds, the opponent takes 1 wound and 1 wound die in piercing damage.
+Magically shape a small piece of wood into a javelin. The javelin will then be thrown with wind magic at the chosen target. Make a spellcasting roll against the target's armor. If the attack succeeds, the opponent takes 1 wound, adding a wound die on a critical.
```

Comment:
I think having the extra wound as a critical keeps the damage buff, while keeping it more in line with the long range, auto-fire spell you were thinking of.